### ISSUE TITLE
Provide `toJS` and `toJSON` functions

### ISSUE THREAD

**Issue by pbomb** [pbomb/flow-immutable-models#15]
In order to support serialization, provide a `toJS` (and `toJSON` alias) function to convert the ImmutableModel into its modelType form.

**Comment by Fi3** [comment]
Why didn't you add `toJSON`? I think that `toJSON` would be a lot faster than `toJS` to encode the model as a string. Is that right?

**Comment by Fi3** [comment]
Sorry just noted that you can pass the object to `JSON.stringify`. Maybe is better have the same interface of immutable.js and implement `toJSON` also here? What do you think?

**Comment by pbomb** [comment]
Yeah, you're right. I'll put together a pull request to add `toJSON` as a shallow operation which will work better when using `JSON.stringify`

**Comment by pbomb** [comment]
I'm going to hold off on this until Immutable-JS 4.0.0 is released. The current version doesn't have correct flow types for the `toJSON` function.
